pages/disability/how-to-file-claim/supplemental-forms.md

```diff
@@ -108,7 +114,10 @@ In some cases, you may need to turn in one or more additional forms to support y
    <td><strong>Authorizing the release of your medical records to VA</strong></td>
     <td><p>Authorization to Disclose Information to VA (VA Form 21-4142)</p>
         <p><a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf">Download VA Form 21-4142</a></p>
-     </td>
+        <p><strong>or</strong></p>
+        <p>You can fill out VA Form 21-4142 when you file a disability claim online.</p>
+        <p><a href="https://www.va.gov/disability/how-to-file-claim/">File a disability compensation claim</a></p> 
```

Comment:
I think `https://www.va.gov` can be omitted from these links, so that our dev/staging environments link properly.